skipper.go

```diff
@@ -368,6 +368,10 @@ type Options struct {
 	// what the []string should contain.
 	DataClientPlugins [][]string
 
+	// MultiPlugins combine multiple types of the above plugin types in one plugin (where
+	// necessary because of shared data between e.g. a filter and a data client).
+	MultiPlugins [][]string
```

Comment:
If we consider the multi plugin the generic form of plugins, we could also just call this option simply: `Plugins`. What do you think?